### ISSUE TITLE
Add a code block for initializing env vars before imports

### ISSUE THREAD

**Issue by NickBolles** [feathers-plus/generator-feathers-plus#107]
### Steps to reproduce
Create a new app
Set a different config directory (such as compiling typescript to a ./dist/server folder, and copying config to ./dist/

### Expected behavior
Should be able to initialize the NODE_CONFIG_DIR before importing @feathers/configuration

```
process.env['NODE_CONFIG_DIR'] = path.join(__dirname, '../', 'config/')
``` 

### Actual behavior
There is no code block to do this before importing stuff

### Suggested fix
Add a code block after the first block of imports in app.ts, but before the feathers blocks for env vars

```
import * as path from 'path';
import favicon from 'serve-favicon';
import compress from 'compression';
import cors from 'cors';
import helmet from 'helmet';
import logger from './logger';

// !code init_env_vars
process.env['NODE_CONFIG_DIR'] = path.join(__dirname, '../', 'config/')
// !end

import feathers from '@feathersjs/feathers';
import configuration from '@feathersjs/configuration';
import express from '@feathersjs/express';
import socketio from '@feathersjs/socketio';
```

**Comment by NickBolles** [comment]
Another option is at the beginning of the index.ts file before importing app. Maybe both makes sense?

**Comment by eddyystop** [comment]
Code insertion point `preface` added to src/index.js and src/app.js.

Published in cli+ v0.7.65 which runs generator v0.6.63.
